packages/tlinks/src/utils/is-farcaster-frame-url.ts

```diff
@@ -0,0 +1,5 @@
+export const isFarcasterFrameUrl = (actionUrl: URL | string) => {
```

Comment:
is the expected? only this single url will be supported?

Reply:
Some new url's will be added in the future. I added the localhost for test.